pkg/storage/etcd/testing/utils.go

```diff
@@ -68,9 +107,14 @@ func configureTestCluster(t *testing.T, name string) *EtcdTestServer {
 		t.Fatal(err)
 	}
 
-	cln := newLocalListener(t)
+	// This is a hack to get the location of certificate files.
+	_, filename, _, _ := runtime.Caller(1)
+	m.CertFile = path.Join(path.Dir(filename), "../etcd_certificates/etcdcert.pem")
```

Comment:
It might be easier to just embed the certs in go source files, instead of loading from disk.